.github/workflows/push-to-azure.yml

```diff
@@ -27,4 +27,6 @@ jobs:
           export DATABRICKS_AAD_TOKEN=$(az account get-access-token --resource 2ff814a6-3304-4ab8-85cb-cd0e6f879c1d | jq .accessToken --raw-output)
           databricks configure --aad-token --host ${{ secrets.DATABRICKS_HOSTNAME }}
           dbfs rm --recursive dbfs:/pipeline
-          dbfs cp pipeline --recursive dbfs:/pipeline
\ No newline at end of file
+          dbfs cp pipeline --recursive dbfs:/pipeline
+          dbfs cp utils.py dbfs:/utils.py --overwrite
+          dbfs cp logging.conf dbfs:/logging.conf --overwrite
```

Comment:
I think if we're going to be doing much more copying to the databricks workspace than we already are then we could do something like `cp . --recursive dbfs:/`, or package it up in a different way, but for now I think this is fine